classes/locations.php

```diff
@@ -51,6 +51,12 @@ public function register_locations() {
 			);
 		}
 
+		// Rewrite rules if we've changed the location permalink slug.
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->